services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/appconfig/validation/RiskScoreClassificationValidator.java

```diff
@@ -81,7 +81,7 @@ private void validateLabel(String label) {
   }
 
   private void validateRiskScoreValueBounds(int value) {
-    if (!RiskScoreValidator.isInBounds(value)) {
+    if (value < 0 || value > ParameterSpec.TOTAL_RISK_SCORE_MAX) {
```

Comment:
Consider introducing `ParameterSpec.TOTAL_RISK_SCORE_MIN` for consistency with `ParameterSpec.TOTAL_RISK_SCORE_MAX` and `ParameterSpec.RISK_SCORE_MIN`